source/language/delays.rst

```diff
@@ -125,8 +125,11 @@ these side effects. Also contrary to TeX, we prohibit overlapping gates.
 Operations on durations
 -----------------------
 
-We can add/subtract two durations, or multiply them by a constant, to get new
-duration. The result must be positive. These are compile time operations since ultimately all
+We can add/subtract two durations, or multiply them by a constant, to get a new
+duration. Division by compile time constants (either ``duration``, ``float``, or ``const``) results 
+in a machine-precision float (see :ref:`divideDuration`). Negative durations are allowed, however
```

Comment:
I had thought it was that `duration / duration` was a float (since the dimensions are `time / time = 1`), but `duration / float` would be a `duration` (as `time / 1 = time`)?